crates/core_simd/src/masks/bitmask.rs

```diff
@@ -78,17 +78,16 @@ where
 {
     #[inline]
     #[must_use = "method returns a new mask and does not mutate the original value"]
-    pub(crate) fn splat(value: bool) -> Self {
-        let mut mask = <LaneCount<N> as SupportedLaneCount>::BitMask::default();
-        if value {
-            mask.as_mut().fill(u8::MAX)
-        } else {
-            mask.as_mut().fill(u8::MIN)
-        }
-        if N % 8 > 0 {
-            *mask.as_mut().last_mut().unwrap() &= u8::MAX >> (8 - N % 8);
-        }
-        Self(mask, PhantomData)
+    #[rustc_const_unstable(feature = "portable_simd", issue = "86656")]
```

Comment:
Since this has never been stable, what's the attribute for?

Reply:
I'm not 100% sure why, but it's required. My guess is since it's a constructor for an unstable type instead of just a method on it.